src/fn/edit-contacts.js

```diff
@@ -0,0 +1,169 @@
+const minimist = require('minimist');
+const environment = require('../lib/environment');
+const fs = require('../lib/sync-fs');
+const path = require('path');
+const { warn, info } = require('../lib/log');
+const pouch = require('../lib/db');
+const toDocs = require('./csv-to-docs');
+
+module.exports = ()=> {
+  const args = parseExtraArgs(environment.pathToProject, environment.extraArgs);
+  const db = pouch();
+  const docDirectoryPath = args.docDirectoryPath;
+  fs.mkdir(docDirectoryPath);
+  const saveJsonDoc = doc => fs.write(`${docDirectoryPath}/${doc._id}.doc.json`, toDocs.toSafeJson(doc) + '\n');
+
+  const csvDir = `${environment.pathToProject}/csv`;
+  if(!fs.exists(csvDir)) {
+    warn(`No csv directory found at ${csvDir}.`);
+    return Promise.resolve();
+  }
+
+  return fs.recurseFiles(csvDir)
```

Comment:
It looks like you are iterating over every .csv file in the "csv" folder and throwing an error when there is at least one file that doesn't start with "contact.".
Is your intent is to support having multiple CSV files be processed at once: `contact._1.csv`, `contact.s.csv`, `contacts._223.csv` ? 
The code would be much cleaner if you just read a `contact.csv` file and process it - just like your example from the documentation - and run the command multiple times over different files, if needed.

Reply:
I agree. We now read one file at a time instead of `recurseFiles` and defaults to `contact.csv` if not specified.
We've also allowed users to specify a list of comma-separated files they would wish to be processed the same way through `--files=contact.1.csv, contact.2.csv` and process them at once.